astro/src/content/docs/lifecycle/migrate-users/bulk/stytch.mdx

```diff
@@ -0,0 +1,382 @@
+
+There are two other files linked to this scrypt plugin that you shouldn't need to alter:
+- `fusionauth-contrib/Password Hashing Plugins/src/test/java/com/mycompany/fusionauth/plugins/ExampleStytchScryptPasswordEncryptorTest.java`, which holds unit tests for the hasher.
+- `fusionauth-contrib/Password Hashing Plugins/src/main/java/com/mycompany/fusionauth/plugins/guice/MyExampleFusionAuthPluginModule.java`, which makes the hasher file known to FusionAuth.
+
+If you make another hash plugin, you will need to make your own test and add your plugin to the Guice file.
+
+Build the Java plugin and add it to FusionAuth:
+- Open a terminal and run the code below to build a Docker container for Java, with the repository shared as a volume into the container.
+  ```sh
+  cd fusionauth-contrib/.devcontainer
+  docker build -t javaimage .
+  cd ..
+  docker run -it --name javabox -v .:/workspace javaimage
+  ```
+- In the Docker container terminal that is now running, run the code below.
+  ```sh
+  cd "/workspace/Password Hashing Plugins"
+  mvn clean install
+  exit
+  ```
+- If you have Java installed locally, you can run the Maven command without Docker.
+
+All tests should pass, and the plugin file should be available in `Password Hashing Plugins/target/fusionauth-example-password-encryptor-0.1.0.jar`.
+
+### Set Up FusionAuth And Deploy The Plugin
+
+Now copy `fusionauth-example-password-encryptor-0.1.0.jar` to your FusionAuth `plugins` directory and restart FusionAuth.
+
+If you are not already running FusionAuth or want to test this process on another instance, you can start FusionAuth in Docker.
+- Open a new terminal in the `fusionauth-import-scripts` directory and run the code below.
+  ```sh
+  cd stytch/fusionAuthDockerFiles
+  docker compose up
+  ```
+- FusionAuth will now be running and browsable at `http://localhost:9011`. You can log in with `admin@example.com` and `password`. The container is called `fa`.
+- In the `fusionauth-contrib` terminal where you built the plugin, run the commands below to copy the JAR file into the FusionAuth container `plugins` directory.
+  ```sh
+  docker exec fa mkdir /usr/local/fusionauth/plugins
+  docker cp "Password Hashing Plugins/target/fusionauth-example-password-encryptor-0.1.0.jar" fa:/usr/local/fusionauth/plugins/fusionauth-example-password-encryptor-0.1.0.jar
+  ```
+- Finally, restart FusionAuth for it to detect the plugin. In the terminal where FusionAuth is running in Docker, press Ctrl + C to stop it, wait, and run `docker compose up` again.
+
+### Save The User Details And Hash To FusionAuth
+
+Now you have the users file `users.json` and the scrypt plugin is installed. To import the users into FusionAuth, you need to run the Node.js import script.
+
+Open a terminal in the `fusionauth-import-scripts` directory and run the code below.
+```sh
+cd stytch/js
+npm update
+node 4_import.mjs
```

Comment:
I am getting the following errors when running this script with the example [users.json](https://github.com/FusionAuth/fusionauth-import-scripts/blob/master/stytch/exampleData/5_userDetailAndHashPreparation/users.json) file. The api key seems good.

![Screenshot 2024-03-14 at 5 56 55 PM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/85caf694-ff4e-4723-a40d-2860e7f73e98)

Reply:
PEBCAK here.  Rebooted and running.

Reply:
I am using the right version, I was just waiting to approve until I could run through and verify that it was working.